static/app/components/featureBadge.stories.tsx

```diff
@@ -33,8 +33,8 @@ export default storyBook(FeatureBadge, story => {
         When using an indicator you might want to position it manually using{' '}
         <kbd>styled(FeatureBadge)</kbd>.
       </p>
-      <Matrix
-        component={props => (
+      <Matrix<typeof FeatureBadge>
+        render={props => (
```

Comment:
Maybe in this case it's nicer to specify it in the template parameter tho

Reply:
can you give an example of what you mean?